types/airtable/index.d.ts

```diff
@@ -73,6 +73,7 @@ declare global {
         }
 
         type Response<TFields> = ReadonlyArray<Row<TFields>>;
+        type ResponseSingle<TFields> = Readonly<Row<TFields>>;
```

Comment:
Why is the single response a `Readonly<Row<...>>`, but an array response is only a `Row<...>`? `ReadonlyArray` is not recursively read-only, it only makes the *array* readonly, not its contents.

Reply:
You're right. I updated the interface names and the definitions as I think it was confusing before.